nmap/plan.sh

```diff
@@ -9,14 +9,55 @@ pkg_source=https://nmap.org/dist/${pkg_name}-${pkg_version}.tar.bz2
 pkg_shasum=a8796ecc4fa6c38aad6139d9515dc8113023a82e9d787e5a5fb5fa1b05516f21
 pkg_deps=(
   core/glibc
-  core/zlib
-  core/nghttp2
   core/gcc-libs
   core/openssl
+  core/pcre
+  core/zlib
 )
 pkg_build_deps=(
+  core/bzip2
   core/coreutils
+  core/diffutils
+  core/file
   core/gcc
+  core/inetutils
+  core/lua
   core/make
+  core/openssh
   core/pkg-config
+  core/readline
+  core/which
 )
+
+do_prepare() {
+  export CFLAGS="${CFLAGS} -O -Wcpp"
+  export CXXFLAGS="${CXXFLAGS} -O -Wcpp"
+
+  if [[ ! -r /usr/bin/file ]]; then
+    ln -sv "$(pkg_path_for file)/bin/file" /usr/bin/file
+    _clean_file=true
+  fi
+}
+
+do_build() {
+  ./configure --prefix="${pkg_prefix}" \
+    --without-zenmap \
+    --with-libpcap=included \
+    --with-libpcre="$(pkg_path_for "core/pcre")" \
+    --with-libz="$(pkg_path_for "core/zlib")" \
+    --with-libssh2="$(pkg_path_for "core/openssh")" \
```

Comment:
I don't think core/openssh provides libssh2.  Rather I believe that is a separate library that isn't yet packaged: https://www.libssh2.org/

```
[30][default:/src:0]# /hab/pkgs/core/nmap/7.60/20171211220813/bin/nmap -V

Nmap version 7.60 ( https://nmap.org )
Platform: x86_64-unknown-linux-gnu
Compiled with: liblua-5.3.4 openssl-1.0.2l nmap-libssh2-1.8.0 libz-1.2.8 libpcre-8.38 nmap-libpcap-1.7.3 nmap-libdnet-1.12 ipv6
```

Note it is using  nmap-libssh2-1.8.0.

The core/openssh package is used however in the tests.

Reply:
yep, true, I had noticed for testing, but didn't pay attention to libssh2. I will package it then. Should we wait for `core/libssh2` or should I modify this to use `included` so we can merge before?

Reply:
@rsertelon Either works for me.  I guess I would lean towards the latter since the package works without waiting on core/libssh2.